.kokoro

```diff
@@ -174,6 +174,26 @@ run_cocoapods() {
   bash <(curl -s https://codecov.io/bash)
 }
 
+generate_website() {
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    # Move into our cloned repo
+    cd github/repo
```

Comment:
Should you `cd ..` after this  to restore the previous `pwd`?  I see two directory changes in this function, but only one that goes back to the parent directory.

Reply:
This cd is for the CI server, which starts not in the repo's root. We generally assume that our scripts are run from the repo's root.

Reply:
Sure, but shouldn't we leave things the way we found them?